components/pdfviewer/accessibility/wai-aria-support.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Wai-Aria Support
+page_title: Telerik UI for Blazor PDFViewer Documentation | PDFViewer  Accessibility
+description: "Get started with the Telerik UI for Blazor PDFViewer and learn about its accessibility support for WAI-ARIA, Section 508, and WCAG 2.1."
+tags: telerik,blazor,accessibility,wai-aria,wcag
+slug: pdfviewer-wai-aria-support 
+position: 50 
+---
+
+# Blazor PDFViewer Accessibility
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+
+
+Out of the box, the Telerik UI for Blazor PDFViewer provides extensive accessibility support and enables users with disabilities to acquire complete control over its features.
+
+
+The PDFViewer is compliant with the [Web Content Accessibility Guidelines (WCAG) 2.1  AA](https://www.w3.org/TR/WCAG21/) standards](https://www.w3.org/TR/WCAG21/) and [Section 508](http://www.section508.gov/) requirements, follows the [Web Accessibility Initiative - Accessible Rich Internet Applications (WAI-ARIA)](https://www.w3.org/WAI/ARIA/apg/) best practices for implementing the [keyboard navigation](#keyboard-navigation) for its `component` role, provides options for managing its focus and is tested against the most popular screen readers.
+
+## WAI-ARIA
+
+
+This section lists the selectors, attributes, and behavior patterns supported by the component and its composite elements, if any.
+
+
+The PDFViewer component contains two inner elements - a toolbar and a page container.
+
+[ToolBar accessibility specification]({{toolbar_a11y_link}})
```

Comment:
Is this link related to the table below? I'm struggling to understand why it's placed like this between the previous sentence and the table below. Consider using this anchor as a part of a complete sentence.

Reply:
This PR only triggers the autogeneration of this article and causes placeholders in some places. It will be fixed.